_puppeteers/bastelfreak.md

```diff
@@ -0,0 +1,22 @@
+---
+layout: profile
+name: Tim Meusel
+image: https://avatars0.githubusercontent.com/u/341866?v=4
+bio: @virtapi Stack founder, I do things with Arch Linux, and Puppet on @voxpupuli
+---
+
+# Tim Meusel
+
+Tim (bastelfreak) Meusel works as a DevOps Engineer for GoDaddy EMEA in Cologne,
+Germany where he develops and maintains a big public cloud platform. Tim is the
+driving force behind various open source solutions at work. He founded the
+VirtAPI-Stack and is a very active Vox Pupuli Maintainer and
+[Project Management Committee](https://github.com/voxpupuli/plumbing/blob/master/share/governance.md#vox-pupuli-governance)
+member. Tim has been doing work in the DevOps area since 2009 and is persuing
+Puppet solutions since 2012. Recently he was reelected to serve on the Vox
+Pupuli Project Management Committee and often speaks at conferences about
+DevOps or Vox Pupuli. He enjoys good BBQ and ice hockey.
+
+bastelfreak
```

Comment:
@binford2k have those last three lines a special meaning?

Reply:
Nope. It just snarfs data from your GitHub profile and dumps it into the template. Everything below the front matter is just free text. Want to remove it?

Reply:
I cleaned it up